### PR TITLE
Set default flake attempt to 1 (not 2)

### PR DESCRIPTION
**Pull request by dims**
Looks like a vestige from way long back! ( see https://github.com/kubernetes/kubernetes/issues/68091 for context )

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
also see https://github.com/kubernetes/test-infra/pull/25924

**Comment by aojea** [comment]
/lgtm
masking flakes is not good, that doesn't help anybody, is just a backdoor for introducing new subtle bugs
/cc @liggitt

File: test/e2e_node/conformance/build/Dockerfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ ENV FOCUS="\[Conformance\]" \
 	   SKIP="\[Flaky\]|\[Serial\]" \
 	   PARALLELISM=8 \
 	   REPORT_PATH="/var/result" \
-	   FLAKE_ATTEMPTS=2 \
```
Comment (249043822) [inline test/e2e_node/conformance/build/Dockerfile:35]: /lgtm

**Pull request merged**